_includes/shared/markup/ccloud/ccloud-setup-self.adoc

```diff
@@ -0,0 +1 @@
+From within the Confluent Cloud UI, click on `Learn` and follow the in-product instructions to launch Kafka and Schema Registry [and ksqlDB] or use the Confluent Cloud CLI.
```

Comment:
1. Should `Confluent Cloud` hyperlink to cloud?
2. Strip `[and ksqlDB]` -- that is only for ksqlDB tutorials
3. Remove `or use the Confluent Cloud CLI` -- because we are leaning on in-product UI tutorials

Reply:
ah yes added in commit https://github.com/confluentinc/kafka-tutorials/pull/786/commits/dbd5f9a6acd43894db949129c6c17167a57f7e39